frontend/package.json

```diff
@@ -27,7 +24,7 @@
     "query-string": "7.1.1",
     "react": "17.0.2",
     "react-app-polyfill": "3.0.0",
-    "react-bootstrap": "2.1.2",
+    "react-bootstrap": "2.5.0-beta.1",
```

Comment:
@BradyMitch Should we be using v2.5 instead of beta?

Reply:
I could try again but I encountered an issue with Tooltips in 2.5.0 that wasn't in the beta